File: programme/merge-two-strings/mergeStrings.go
// Write a programme to merge two strings

package main

import "fmt"

func main() {
	var str1, str2 string

	fmt.Print("Enter String 1 : ")
	if _, err := fmt.Scan(&str1); err != nil {
		panic("Scan Error str1")
	}

	fmt.Print("Enter String 2 : ")
	if _, err := fmt.Scan(&str2); err != nil {
		panic("Scan Error str2")
	}

	mergedString := str1 + str2

	fmt.Printf("Output: %s\n", mergedString)
}

File: programme/merge-two-strings/mergeStrings.cpp
// Write a programme to merge two strings
#include <iostream>

using namespace std;

int main()
{
    string str1, str2;

    cout << "Enter String 1 : ";
    cin >> str1;
    cout << "Enter String 2 : ";
    cin >> str2;

    string mergedString = str1 + str2;
    cout << "\nMerged String : " << mergedString << endl;
    return 0;
}
